bionic/flow.py

```diff
@@ -1557,6 +1557,19 @@ def ...
 
         return flow
 
+    def reloading_in_place(self):
```

Comment:
Let's call this `reload`. Also, I'd suggest making this method the primary way of accessing this functionality -- that is, rather having this method's docstring refer to `reloading` for details, `reloading`'s should refer to this one. Similarly, we can make this method the first one to appear in the docs.

It might make sense to go as far as deprecating `reloading`, although we can always do that later.